messaging/copr_messaging/private/hierarchy.py

```diff
@@ -39,6 +39,16 @@ def __str__(self):
     def _str_prefix(self):
         return "Copr Message"
 
+    @property
+    def app_name(self):
+        """
+        Return the name of the application that generated the message.
+
+        Returns:
+            the name of the application (copr)
+        """
+        return "copr"
```

Comment:
This is supposed to be human-readable, so you can capitalize it.

Reply:
@ryanlerch can you please update?

Reply:
I eventually updated.